lib/commands/create.dart

```diff
@@ -114,7 +130,6 @@ class TizenCreateCommand extends CreateCommand {
     final String appType = stringArg('tizen-language');
     // The Dart plugin template is not currently supported.
     const String pluginType = 'cpp';
```

Comment:
You may want to rename these variables to `appLanguage` and `pluginLanguage` to apply the above suggestion.